src/components/matches/questionnaire.component.js

```diff
@@ -95,6 +86,14 @@ export default function Questionnaire(props) {
     personalityType: "",
   });
 
+  useEffect(() => {
+    if (!loginInfo.userInfo.session) {
+      alert("Please log in to view your profile.");
+      return props.history.push("/login");
+    }
+    window.scrollTo(0, 0);
+  }, []);
```

Comment:
![image](https://user-images.githubusercontent.com/66970460/108099362-5a939400-7039-11eb-97b2-8ecda3c1251b.png)

Might have to add these two to the `useEffect` dependency? If they're not needed we could also just throw in `// eslint-disable-next-line react-hooks/exhaustive-deps` before the line to disable the error as well.